sandbox/auth/internal/pcke/pcke.go

```diff
@@ -0,0 +1,68 @@
+// Code in this file taken from https://go-review.googlesource.com/c/oauth2/+/463979
+// under a BSD license.
+//
+// The standard go library for oauth2 does not support PCKE, but a proposal
+// has been accepted to add it in a future version of go. We copy the code from
+// the proposal so that it's easy to switch to the standard library when it's
+// available.
+//
+// Issue: https://github.com/golang/oauth2/issues/603
+// Proposal: https://github.com/golang/go/issues/59835
+
+package pcke
```

Comment:
I think needs to be pkce :)
https://go-review.googlesource.com/c/oauth2/+/463979